### PR TITLE
question answering using question generation matching

### PR DESCRIPTION
**Pull request by maxliuofficial**
Updated question answering module to include question to question matching (depends on question generation tuple output).

### CODE REVIEW

**Comment by Aklaran** [comment]
Looks good to me, but you'll have to resolve the merge conflicts before merging

File: answer_questions.py [hunk 1]
```diff
@@ -28,26 +28,54 @@ def __init__(self, text_file, question_file):
...
+        print('STOPWORDS REMOVED: ', question)
 
-        # this sets up the questions
-        q_file = open(question_file,"r+")
-        self.questions = q_file.read()
-        #self.questions = self.questions.decode('utf8').upper().encode('ascii')
-        self.q_list = self.questions.split('\n')
+        # generate (similarity, question) list for each question
+        candidates = []
+        for (sent, q_class, correct_answer) in self.wh_questions:
```
Comment (Aklaran) [inline answer_questions.py]: @amyzhang17 this is the line defining the tuple

Comment (Aklaran) [reply answer_questions.py]: @max you could just revise to this:
```suggestion
        for question in self.wh_questions:
            sent = question.q_string
            q_class = question.q_class
            correct_answer = question.q_answer
```

Comment (maxliuofficial) [reply answer_questions.py]: Yeah, that looks good I'll make that change.

**Comment by maxliuofficial** [comment]
@Aklaran do you want me to merge it or are you going to? I don't have any experience with conflict resolution tools.

**Pull request merged**